src/tooltips.tsx

```diff
@@ -120,6 +146,7 @@ interface TooltipComponentState {
 class TooltipComponent extends React.Component<{}, TooltipComponentState> {
 	private tooltipElement: HTMLDivElement;
 	private tooltipElementInner: HTMLDivElement;
+	private observer: MutationObserver;
```

Comment:
This will hold no object initially, so let's type it
```suggestion
	private observer: MutationObserver | undefined;
```

Reply:
:+1: